example/README.md

```diff
@@ -0,0 +1,16 @@
+# example
+
+A new Flutter project.
```

Comment:
can we improve this readme? just a single line saying it's an example project for the splash screen shall suffice.